src/sonic-host-services/tests/procdockerstatsd_test.py

```diff
@@ -36,9 +36,13 @@ def test_convert_to_bytes(self):
             ('500MiB', 524288000),
             ('66.41MiB', 69635932),
             ('333.6MiB', 349804954),
+            ('7GB', 7000000000),
+            ('800GB', 800000000000),
             ('1GiB', 1073741824),
             ('500GiB', 536870912000),
-            ('7.751GiB', 8322572878)
+            ('7.751GiB', 8322572878),
+            ('8.1TiB', 8906000000000000),
+            ('12TB', 12000000000000)
```

Comment:
Suggest to also add at least one TB test case with a decimal point, following my existing pattern.